cmd/control/os.go

```diff
@@ -216,6 +216,16 @@ func startUpgradeContainer(image string, stage, force, reboot, kexec bool, upgra
 		}
 	}
 
+	fmt.Printf("Upgrading to %s\n", image)
+	confirmation := "Continue"
+	imageSplit := strings.Split(image, ":")
```

Comment:
In general we should start using the distribution code to parse image names because there's a tons of gotchas.  Right now this is fine though.